mintapi/api.py

```diff
@@ -265,18 +266,15 @@ def get_transactions_json(self, include_investment=False,
                         else 'task=transactions,txnfilters&filterType=cash'))
             result = self.request_and_check(
                 url, headers=self.json_headers,
-                expected_content_type='text/json')
+                expected_content_type='text/json|application/json')
             data = json.loads(result.text)
             txns = data['set'][0].get('data', [])
-            df = pd.DataFrame(txns)
             if start_date:
-                dates = list(df['odate'])
-                last_dt = self._dateconvert(dates[-1])
+                last_dt = self._dateconvert(txns[-1]['odate'])
                 if last_dt < start_date:
-                    keep_txns = []
-                    for item in txns:
-                        if self._dateconvert(item['odate']) >= start_date:
-                            keep_txns.append(item)
+                    keep_txns = [
+                        item for item in txns
+                        if self._dateconvert(item['odate']) >= start_date]
```

Comment:
Oh, yes I'm very familiar with them, I just didn't see the square brackets above so I thought 276 was its own line. Now I understand what is happening syntactically, thanks! :)